src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -501,7 +505,7 @@ public static boolean postOMFBatch(String payload) {
             OSIsoftConfig.getOmfPostHeaders() + postHeaderType,
             payload,
             "",
-            responseFilename); // data
+            responseDirectory + "data" + responseFileName); // data

```

Comment:
why is "data" in the path?

Reply:
these response files are just for the data messages and it will separate out the response file on the flexy so OSIsoft error messages will be saved for data message errors. It's really hard to debug issues without separate response files that are saved.

Reply:
I think the containers and data filename prefixes should be documented then (static final string w/ javadoc).